src/feature/Client/components/OrderItem.js/OrderItem.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import OrderDetail from "./OrderDetail";
+import "./OrderItem.scss";
+
+function OrderItem(props) {
+  const { order } = props;
+  const renderCart = (list) => {
+    return list.map((item, index) => {
+      return <OrderDetail product={item} key={index} />;
+    });
+  };
+
+  const renderStatus = (status) => {
+    let orderStatus = "";
+    switch (status) {
+      case "pending":
+        orderStatus = "Chờ xác nhận";
+        return orderStatus;
```

Comment:
```suggestion
    switch (status) {
      case "pending":
        return "Chờ xác nhận";
```
Hoặc có thể return ra tên key json (VD: status.pending) để áp dụng i18n